### ISSUE TITLE
operator ++ Exception

### ISSUE THREAD

**Issue by AndreyZM** [bridgedotnet/Bridge#1812]
```
System.InvalidCastException: Unable to cast object of type 'ICSharpCode.NRefactory.Semantics.ErrorResolveResult' to type 'ICSharpCode.NRefactory.Semantics.OperatorResolveResult'.
```

### Steps To Reproduce

http://deck.net/9a379a503bcf8aec73441ae0331d549c

```csharp
public class _Object
{
    static public implicit operator _Object (int d)
    {
        return null;
    }
    
    static public _Object operator +(_Object left,_Object right)
    {
        return null;
    }
    
    static public implicit operator int (_Object d)
    {
        return 0;
    }
    
    void a()
    {
        _Object a = null;
        a++;
    }
}
```

**Comment by geoffreymcgill** [comment]
HI @AndreyZM - Thanks for the report. We will investigate and try to fix right away. 

Here's a Deck with your code sample demonstrating how to reproduce:
http://deck.net/9a379a503bcf8aec73441ae0331d549c

**Comment by vladsch** [comment]
Thanks for the report, we will try to fix it. The workaround to override ++ operator
```cs

**Comment by geoffreymcgill** [comment]
This issue has been fixed and will be included in the Bridge 15.0.1 release being pushed to NuGet in a few minutes. 

Your original Deck now compiles properly. 

Thanks for reporting the issue.
